client/src/main/webjar/organization-list/organization-list-route.js

```diff
@@ -0,0 +1,31 @@
+(function (angular) {
```

Comment:
Shouldn't this file be named organization-list.route.js? (dot instead of hyphen before the route)